demisto_sdk/validation/file_validator.py

```diff
@@ -244,214 +253,45 @@ def is_valid_release_notes(self, file_path):
         if not release_notes_validator.is_file_valid():
             self._is_valid = False
 
-    def validate_modified_files(self, modified_files):  # noqa: C901
-        """Validate the modified files from your branch.
-
-        In case we encounter an invalid file we set the self._is_valid param to False.
-
-        Args:
-            modified_files (set): A set of the modified files in the current branch.
-        """
-        for file_path in modified_files:
-            old_file_path = None
-            if isinstance(file_path, tuple):
-                old_file_path, file_path = file_path
-
+    def validate_files(self, affected_files, is_added=False):
+        REGEX_TO_VALIDATOR = {
+            TEST_PLAYBOOK_REGEX: lambda x: print('Not validating Test playbook'),
+            YML_INTEGRATION_REGEXES: self.validate_integration_yml,
+            YML_BETA_INTEGRATIONS_REGEXES: self.validate_beta_integration,
+            IMAGE_REGEX: self.validate_image,
+            REPUTATION_REGEX: self.validate_reputation_file,
+            SCRIPT_REGEX: self.validate_script,
+            PACKAGE_SCRIPTS_REGEXES: self.validate_script,  # todo: check this out
+            PLAYBOOKS_REGEXES_LIST: self.validate_playbook,
+            JSON_INDICATOR_AND_INCIDENT_FIELDS: self.validate_indicator_and_incident_fields,
+            JSON_ALL_LAYOUT_REGEXES: self.validate_layout
```

Comment:
add comment for un-tested files:
Classifiers,
```suggestion
            JSON_ALL_LAYOUT_REGEXES: self.validate_layout,
            # TODO
            # Dashboards,
            # Incident Types,
            # Reports, 
```